packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -21,14 +22,26 @@ export function useFrequentlyBoughtTogether<TObject>({
   recommendClient,
   threshold,
   transformItems: userTransformItems,
-}: UseFrequentlyBoughtTogetherProps<TObject>) {
-  const [result, setResult] = useState<GetRecommendationsResult<TObject>>({
-    recommendations: [],
-  });
-  const { status, setStatus } = useStatus('loading');
+  initialResults: userInitialResults,
```

Comment:
does this still need to be renamed, it could be this right (with the usage changed too)?

```suggestion
  initialResults,
```

Reply:
✅ : https://github.com/algolia/recommend/pull/109/commits/fe195d38f2074d588c9e2549d77260eb3360bf3c

got a const here that is used to stabilize the initialResults and initialize the useState here: 
```
const initialResults = useStableValue<GetRecommendationsResult<TObject>>({
    recommendations: [],
    ...userInitialResults,
  });
  const [result, setResult] = useState<GetRecommendationsResult<TObject>>(
    initialResults
  );
  ```
  
  but i also need the initialResults from the user here, hence the renaming:
  ```
  const shouldFetch = !userInitialResults || !isFirstRenderRef.current;
  ```